rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -4803,6 +4804,104 @@ public void addSignatureMultipleInputsPartiallyValid() throws Exception {
         }
     }
 
+    @Test
+    public void processPegIn_no_lock_tx() throws IOException {
+        BridgeSupport bridgeSupport = getBridgeSupport(bridgeConstants, mock(BridgeStorageProvider.class));
+
+        Assert.assertFalse(bridgeSupport.processPegIn(new BtcTransaction(btcParams),
+                mock(Transaction.class),
+                0,
+                mock(Sha256Hash.class)));
+    }
+
+    @Test
+    public void processPegIn_tx_no_lockable_by_invalid_sender() throws IOException {
+        assertProcessPegIn(ConsensusRule.RSKIP143,true, false,
+                BtcLockSender.TxType.P2SHMULTISIG);
+    }
+
+    @Test
+    public void processPegIn_tx_no_lockable_by_not_whitelisted_address() throws IOException {
+        assertProcessPegIn(null,false, false,
+                BtcLockSender.TxType.P2PKH);
+    }
+
+    @Test
+    public void processPegIn_tx_no_lockable_by_surpassing_locking_cap() throws IOException {
+        assertProcessPegIn(ConsensusRule.RSKIP134,true, true,
+                BtcLockSender.TxType.P2PKH);
+    }
+
+    @Test
+    public void processRelease_no_release_tx() throws IOException {
+        BridgeSupport bridgeSupport = getBridgeSupport(bridgeConstants, mock(BridgeStorageProvider.class));
+        Assert.assertFalse(bridgeSupport.processRelease(new BtcTransaction(btcParams), mock(Sha256Hash.class)));
+    }
+
+    @Test
+    public void processMigration_no_migration_tx() throws IOException {
+        BridgeSupport bridgeSupport = getBridgeSupport(bridgeConstants, mock(BridgeStorageProvider.class));
+        Assert.assertFalse(bridgeSupport.processMigration(new BtcTransaction(btcParams), mock(Sha256Hash.class)));
+    }
+
+    private void assertProcessPegIn(@Nullable ConsensusRule consensusRule, boolean isWhitelisted,
```

Comment:
If I'm not wrong, you are testing refunds in here, right?
```suggestion
    private void assertRefundInProcessPegIn(@Nullable ConsensusRule consensusRule, boolean isWhitelisted,
```

Reply:
Done